src/coreclr/vm/reflectioninvocation.cpp

```diff
@@ -823,7 +823,7 @@ FCIMPL5(Object*, RuntimeMethodHandle::InvokeMethod,
 
     LPBYTE pTransitionBlock = pAlloc + TransitionBlock::GetNegSpaceSize();
 
-    CallDescrData callDescrData;
+    CallDescrData callDescrData = {};
```

Comment:
```suggestion
    CallDescrData callDescrData;
```
This should not be needed with the workaround below.

Reply:
I thought I removed that...